### ISSUE TITLE
Cuctom id's for jobs

### ISSUE THREAD

**Issue by abhisek1318** [Automattic/kue#1168]
How to add custom id to jobs ?

**Comment by mrfoh** [comment]
@abhisek1318 Hello did you find a way to do this yet? Also looking for a way to set custom Ids for jobs

**Comment by fredvollmer** [comment]
It would be great to be able to use deterministic/predictable keys, set via a custom id, so that then a job can be removed without needing to know the random/unique id...

**Comment by shahidmau** [comment]
Yes, it is a must-have feature. We can do it in **bee-queue** library by using **setId** function.
